static/app/components/performance/searchBar.tsx

```diff
@@ -24,10 +24,17 @@ export type SearchBarProps = {
   onSearch: (query: string) => void;
   organization: Organization;
   query: string;
+  className?: string;
```

Comment:
I don't see this being used explicitly in the diff, is it implicitly passed down with the styled component definition or should this be deleted?

Reply:
yeah it's passed implicitly by the styled component